contracts/near/eth-client/src/lib.rs

```diff
@@ -574,14 +574,10 @@ impl EthClient {
 
     // check if the author address is valid and is in the validator set.
     #[cfg(feature = "bsc")]
-    fn bsc_is_validator(&self, header: &BlockHeader) -> bool {
+    pub fn bsc_is_validator(&self, header: &BlockHeader) -> bool {
```

Comment:
This is a bad practice to make the function pub just for the tests.
Please turn it back and find a workaround
Refs:
https://doc.rust-lang.org/book/ch11-03-test-organization.html#testing-private-functions
https://stackoverflow.com/questions/27978377/how-do-i-test-private-methods-in-rust

Reply:
Hey Karim!
I'll fix it thx :)

Reply:
Done